tests/playwright/Report.md

```diff
@@ -0,0 +1,176 @@
+ - expect-playwright integration which provides common expect helper functions
+
+ jest-playwright is added to the jest configuration as a preset which makes all the features availbale to be used at your disposal. it was inspired by [jest-puppeteer](https://github.com/smooth-code/jest-puppeteer/blob/master/README.md#start-a-server).
+
+ **Example of jest-playwright.config.js**
+
+```
+    module.exports = {
+        browsers: [process.env.BROWSER || "chromium"],
+        devices: ["iPhone 6", "Pixel 2"]
+    }
+```
+
+Below are some great and useful functions that come with jest-playwright;
+
+Reset helper functions. This functions come in handy when you want to reset the browser or page for various resasons such as deleting cookies. It exposes functions like, `resetPage(), resetContext() and resetBrowser()`.
+Example usage of reset current browser
+
+```
+    beforeEach(async () => {
+        await jestPlaywright.resetBrowser()
+    })
+```
+
+Debug helper functions. This help in debugging test incase there are test failure. It uses the `jestPlaywrightDebug()`
+
+Example usage of Debug
+
+```
+    test.jestPlaywrightDebug('failed', async ({ page }) => {
+        await page.goto('https://www.wikipedia.org/')
+        const title = await page.title()
+        await expect(title).toBe('Nothing')
+    })
+```
+
+For more of these gems that come with jest-playwright, please visit the [Github Repo](https://github.com/playwright-community/jest-playwright)
+
+#### Playwright and WebdriverIO metrics
+
+**Statics on  WebdriverIO and Playwright Popularity**
+
+![Screenshot](statistics.png)
+
+The above data was adopted from [npm trends](https://www.npmtrends.com/cypress-vs-playwright-vs-puppeteer-vs-selenium-vs-testcafe) showing some of the most popular automation test tools in terms of popularity. From the above charts, we see that WebdriverIO still trumps Playwright when it comes to popularity among the testing community.
+
+**WebdriverIO and Playwright Mediawiki-core Tests Analysis**
+
+![Screenshot](evaluation-chart.png)
+
+The above visaulised chart represents data that was collected by running mediawiki/core tests in WebdriverIO and Playwright. A total of 14 tests were run using both frameworks on [Github Actions](https://docs.github.com/en/actions) platform.
+
+The tests were run 40 times in both framework to ascertain stability and reliability. No flakiness and failures were encountered during the tests run. As shown above, there is consitency in time when the tests were run in WebdriverIO and Playwright.
+
+The above chart further shows that Playwright is much faster than WebdriverIO in terms of speed.
+ 
+### Why adapt Playwright
+Playwright offers some great reasons such as;
+- It's simple to set up
+- Stable features. 
+- Ability to install Chrome, Firefox or WebKit (Safari) automatically
+- Bidirectional (events) – automating things like console logs is easy
+- Maintained by Microsoft people with experience maintaining Puppeteer
```

Comment:
Do you have a reference for this? I thought Puppeteer was developed by Google and Playwright by Microsoft.

Reply:
Yes. So the same team that developed Puppeteer moved to Microsoft and developed Playwright.